crates/ibc/src/core/ics04_channel/events.rs

```diff
@@ -1058,3 +1058,149 @@ impl TryFrom<TimeoutPacket> for abci::Event {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use tendermint::abci::Event as AbciEvent;
+
+    #[test]
+    fn ibc_to_abci_channel_events() {
+        struct Test {
+            kind: IbcEventType,
+            event: AbciEvent,
+            expected_keys: Vec<&'static str>,
+            expected_values: Vec<&'static str>,
+        }
+
+        let port_id = PortId::default();
+        let channel_id = ChannelId::default();
+        let connection_id = ConnectionId::default();
+        let counterparty_port_id = PortId::default();
```

Comment:
Let's not use `default()` here too; we can just use `new()`. I don't like the use of `default()` in tests as they make it harder to verify correctness quickly (I need to go check the implementation for `default()` for each of these), for little gain.